html-forms/index.html

```diff
@@ -23,17 +23,17 @@ <h1>Coding Form!</h1>
             </div>
             <div>
                 <label for="radio">Favorite Language (choose one)<br></label>
```

Comment:
This should be just a general text element, such as `p`. For this section, the `label` should actually be the text next to each radio input.